inputfiles/overridingTypes.jsonc

```diff
@@ -3553,6 +3553,11 @@
                     }
                 ],
                 "overrideType": "ReadableStreamDefaultReader<T> | ReadableStreamBYOBReader"
+            },
+            {
+                "name": "AllowSharedBufferSource",
+                // No SharedArrayBuffer in ES5, oh no.
```

Comment:
For now I'm just removing SharedArrayBuffer, as I don't have any better solution. cc @sandersn

Reply:
Uhhh, is the problem here that DOM types have to work with any ES target, even ones that are incredibly old, like ES5/ES2015?

If so, this is going to become more of a problem over time. Maybe we could split the DOM types into parts which require newer ES types?

Reply:
Theoretically we can have `dom.sharedbuffer` like we have `dom.iterable`, yes. That should work for now, until we get any case that function overloads don't work (which is already a problem for `dom.iterable`). Let me file an issue for that...

Reply:
Filed #1579.